validator/constants.py

```diff
@@ -43,9 +43,9 @@
 with open(os.path.join(os.path.dirname(__file__), 'app_versions.json')) as avs:
     APPROVED_APPLICATIONS = json.load(avs)
 
-# Add the '*' fake max version. Used specifically for WebExtensions, see
-# https://github.com/mozilla/amo-validator/issues/390.
+# Add the '*' fake max version for Desktop and Android.
 APPROVED_APPLICATIONS['1']['versions'].append('*')
+APPROVED_APPLICATIONS['7']['versions'].append('*')
```

Comment:
You mean https://addons.mozilla.org/en-US/firefox/pages/appversions/ ? It already has "*" for Desktop, we can easily add it for Android as well in the admin tools.